contracts/src/v0.8/dev/ocr2dr/OCR2DRRegistry.sol

```diff
@@ -795,6 +829,13 @@ contract OCR2DRRegistry is
     }
   }
 
+  modifier onlyAuthorizedUsers() {
+    if (!ORACLE.isAuthorizedSender(msg.sender)) {
```

Comment:
This was discussed via DM, but is documented here for clarity.

What prevents this is `tx.origin` is used to gate sendRequest while `msg.sender` is used to gate createSubscription.

Therefore, in the case described above, the Automation node's wallet would not be calling `createSubscrption`, the Automation node contract would be making the call and that call would fail since the Automation node's contract is not in the allowList.